app/views/CalculationAdditionalView.scala.html

```diff
@@ -22,10 +22,13 @@
     details: govukDetails
 )
 
-@(daysSince31: Long, vatDue: Boolean, additionalPenaltyRate: String , parentCharge: String, amountToDate: BigDecimal)(implicit request: Request[_], messages: Messages, appConfig: AppConfig, user: User[_])
+
+@(daysSince31: Long, vatDue: Boolean, additionalPenaltyRate: String ,startDate:String, endDate:String, parentCharge: String, amountToDate: BigDecimal)(implicit request: Request[_], messages: Messages, appConfig: AppConfig, user: User[_])
 
 @layout(pageTitle = messages("calculation.title.additional"), isPageFullWidth = false) {
-    <h1 class="govuk-heading-xl">@messages("calculation.heading.additional")</h1>
+       <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl">@messages("calculation.caption", startDate, endDate)</span>
```

Comment:
Could you make the class to be govuk-heading-l? Then it matches: https://github.com/hmrc/penalties-frontend/blob/1d1721b6d29134f650a7ffa60eadf7ebf225eb88/app/views/CalculationLPPView.scala.html#L40-L43

Reply:
Done